processor/src/main/java/org/mapstruct/ap/internal/model/Field.java

```diff
@@ -105,4 +105,33 @@ public void setTypeRequiresImport(boolean typeRequiresImport) {
         this.typeRequiresImport = typeRequiresImport;
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
I thought about that.. There's nothing more unique in the mapper class  on that level then the variables. The used mapper is a variable. Type would strictly be not needed.